### ISSUE TITLE
Provide TypeScript types

### ISSUE THREAD

**Issue by simonihmig** [adopted-ember-addons/ember-keyboard#464]
Would be great if the addon would ship with its own `index.d.ts` types! 😀

**Comment by lukemelia** [comment]
Yes, please! I would also be open to authoring this addon in typescript.

**Comment by NullVoxPopuli** [comment]
anyone have some local .d.ts file that we can paste in to the addon's directory? :sweat_smile:
